clouddq/main.py

```diff
@@ -349,7 +349,9 @@ def main(  # noqa: C901
             bigquery_client.assert_dataset_is_in_region(
                 dataset=target_dataset_id, region=gcp_region_id
             )
-            bigquery_client.assert_required_columns_exist_in_table(target_table_ref)
+            bigquery_client.assert_required_columns_exist_in_table(
+                target_bigquery_summary_table
+            )
```

Comment:
This was a bug actually. The table name is `target_bigquery_summary_table`, whereas `target_table_ref` is a `Table` object and it does print the `ALTER TABLE` statement right.

Reply:
Ah great catch thanks!